appendices.tex

```diff
@@ -177,7 +182,7 @@ \section{Prelude to Lie Algebroids}
   Then the triple $(T^*M,[\cdot,\cdot],\rho)$ is a Lie algebroid.
 \end{example}
 \begin{example}
-  Let $L\subset\mathbb{T}M$ be a Dirac structure on $M$ and define $\rho:=\text{pr}|_L$, where $\text{pr}:\mathbb{T}M\to TM$ is the natural projection. Let $[\cdot,\cdot]$ be the restriction of the Courant bracket to $\Gamma(L)$. Then $(L,[\cdot,\cdot],\rho)$ is a Lie algebroid.
+Let $L\subset\mathbb{T}M$ be a Dirac structure on $M$ and define $\rho:=\text{pr}|_L$, where $\text{pr}:\mathbb{T}M\to TM$ is the natural projection. Let $[\cdot,\cdot]_C$ be the restriction of the Courant bracket to $\Gamma(L)$. Then $(L,[\cdot,\cdot]_C,\rho)$ is a Lie algebroid.
```

Comment:
This feels wrong why are you adding the `_C` back?

Reply:
> This feels wrong why are you adding the `_C` back?

Above, I defined a different bracket on the generalised tangent bundle, and in a remark I defined the one called the Courant bracket and called it [ , ]_C. This is the bracket which makes a Dirac manifold into a Lie algebroid, as it's actually a Lie bracket (when restricted). The proof is in Courant's original paper, which is maybe worth citing: https://doi.org/10.1090/S0002-9947-1990-0998124-1

Reply:
Oh indeed, I had missed the second definition in the remark looking at the code. I’ll adda a TODO for the citation and merge

Reply:
```suggestion
% TODO: add reference to https://doi.org/10.1090/S0002-9947-1990-0998124-1
Let $L\subset\mathbb{T}M$ be a Dirac structure on $M$ and define $\rho:=\text{pr}|_L$, where $\text{pr}:\mathbb{T}M\to TM$ is the natural projection. Let $[\cdot,\cdot]_C$ be the restriction of the Courant bracket to $\Gamma(L)$. Then $(L,[\cdot,\cdot]_C,\rho)$ is a Lie algebroid.
```